src/Thinktecture.Relay.Server.Management/ManagementApiConstants.cs

```diff
@@ -0,0 +1,27 @@
+namespace Thinktecture.Relay.Server.Management;
+
+/// <summary>
+///  Provides constant values for the management api defaults.
+/// </summary>
+public static class ManagementApiConstants
```

Comment:
After removing the marked strings, this class should be named `ManagementApiPolicyName`.